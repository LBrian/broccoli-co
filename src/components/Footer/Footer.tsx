function Footer() {
  const year = new Date().getFullYear()

  return (
    <footer className='footer footer-center p-10 short:p-4 bg-secondary text-secondary-content sticky bottom-0 z-10'>
      <div>
        <p className='font-bold'>
          {document.title}
          <br />
          Providing reliable tech since 1992
        </p>
        <p>Copyright Â© {year} - All right reserved</p>
      </div>
    </footer>
  )
}

export default Footer
