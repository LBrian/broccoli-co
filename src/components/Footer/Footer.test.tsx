import { render } from '@testing-library/react'
import Footer from './Footer'

test('renders Footer component', () => {
  expect(render(<Footer />)).toMatchSnapshot()
})
