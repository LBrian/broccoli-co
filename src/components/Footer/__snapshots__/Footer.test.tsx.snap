// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer footer-center p-10 short:p-4 bg-secondary text-secondary-content sticky bottom-0 z-10"
      >
        <div>
          <p
            class="font-bold"
          >
            <br />
            Providing reliable tech since 1992
          </p>
          <p>
            Copyright © 
            2023
             - All right reserved
          </p>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer footer-center p-10 short:p-4 bg-secondary text-secondary-content sticky bottom-0 z-10"
    >
      <div>
        <p
          class="font-bold"
        >
          <br />
          Providing reliable tech since 1992
        </p>
        <p>
          Copyright © 
          2023
           - All right reserved
        </p>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
